.env.example

```
@@ -0,0 +1,4 @@
+NEXT_PUBLIC_BACKEND_URL= 
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)